src/sync/index.js

```diff
@@ -695,7 +694,7 @@ async function getAddressBalances() {
       const getBotBalancePromises = [];
 
       _.map(addressBatches[loop.iteration()], async (address) => {
-        const getBotBalancePromise = new Promise(async (resolve) => {
+        const getBotBalancePromise = new Promise(async (_resolve) => {
```

Comment:
lets name this `getBotBalanceResolve` to be more explicit?

Reply:
updated